src/components/button/button.svelte

```diff
@@ -145,6 +159,17 @@
     [data-is-button-target]:hover :host .leoButton,
     [data-is-button-target]:hover .leoButton {
       --leo-icon-color: var(--icon-hover-color, var(--icon-color));
+      --mixed-primary-color: var(--leo-color-primitive-primary-70);
+
+      /** If we support the color-mix syntax, infer the hover color */
+      @supports (color: color-mix(in srgb, transparent, transparent)) {
+        --mixed-primary-color: color-mix(
+          in srgb,
+          var(--primary-color) 80%,
+          var(--foreground)
```

Comment:
Would it make more sense here to do the following since the "spec" is 20% black or white?

```suggestion
          var(--primary-color),
          var(--foreground) 20%
```